extensions/blocks/podcast-player/style.scss

```diff
@@ -180,13 +181,17 @@ $jetpack-podcast-player-error: $alert-red;
 		margin: 0 0 $gutter-l 0;
 		font-size: $editor-font-size;
 		line-height: $editor-line-height;
-		//crop the description if too long
-		display: -webkit-box;
-		-webkit-line-clamp: 4;
-		-webkit-box-orient: vertical;
-		overflow: hidden;
-		max-height: 105px; //IE11 fallback
 		color: inherit;
+		overflow: hidden;
+		max-height: 4em * $editor-line-height; // IE11 fallback for line-clamp
+
+		@supports ( display: -webkit-box ) {
+			//crop the description if too long
```

Comment:
super minor but can you please format it as other comments? At least a space after `//` but having capital letter and a period at the end to match to WP style would be amazing 💯

Reply:
Absolutely! <3